cmd/cmd.go

```diff
@@ -114,14 +114,18 @@ func SendMSG(cmd *cobra.Command, args []string) {
 	c, _ := lib.NewClient(flagApi, flagUser, key)
 
 	// Query for the recipients public key.
-	rawPubKey, err := c.Get(flagRecip)
-	if err != nil {
-		log.Printf("failed to parse public key")
-		return
+	rawPubKeys := [][]byte{}
+	for _, flagRecip := range flagRecips {
+		rawPubKey, err := c.Get(flagRecip)
+		if err != nil {
+			log.Printf("failed to parse public key")
+			continue
```

Comment:
thoughts on returning early if any of the recip pk's fail to parse?

Reply:
that makes sense - all or nothing is probably better than best effort in this scenario